Monika After Story/game/script-topics.rpy

```diff
@@ -11546,9 +11605,39 @@ label monika_grad_speech_not_paying_attention:
     return
 
 label monika_grad_speech_ignored_lock:
-    #Second timeout, lock speech
     hide screen mas_background_timed_jump
-    #Set false for modified dialogue in the random
+
+    # Still, we make a double check.
```

Comment:
Also a new label.

Reply:
Also understood and [changed](45232dcfa). The label for new code is with a `_doublecheck` now, so the current flow is: 

`monika_grad_speech_call` 
to 
`monika_grad_speech_ignored_doublecheck` 
to(if no response)
`monika_grad_speech_ignored`.